src/ert/ensemble_evaluator/snapshot.py

```diff
@@ -446,34 +398,28 @@ def get_job_status_for_all_reals_and_steps(
     def reals(self) -> Mapping[str, "RealizationSnapshot"]:
         return self._my_partial.reals
 
-    def steps(self, real_id: str) -> Dict[str, "Step"]:
-        return {
-            step_idx[1]: Step(**step_data)
-            for step_idx, step_data in self._my_partial._step_states.items()
-            if step_idx[0] == real_id
-        }
+    def step(self, real_id: str) -> Dict[str, Union[str, datetime.datetime]]:
+        return self._my_partial._step_states[real_id]
 
-    def jobs(self, real_id: str, step_id: str) -> Dict[str, "Job"]:
+    def jobs(self, real_id: str) -> Dict[str, "Job"]:
```

Comment:
To maintain the same API, what about renaming this to `get_jobs`?

Reply:
Fixed